lib/common.sh

```diff
@@ -267,7 +268,11 @@ setGoVersionFromEnvironment() {
 }
 
 determineTool() {
-    if [ -f "${depTOML}" ]; then
+    if [ -f "${goMOD}" ]; then
+        TOOL="gomodules"
+        ver=${GOVERSION:-$(awk '{ if ($1 == "//" && $2 == "+heroku" && $3 == "goVersion" ) { print $4 } }' ${goMOD})}
```

Comment:
Should we exit `awk` after the first match? If users mistaken have multiple `// +heroku goVersion 1.10` there will be multiple returns:

```
awk '{ if ($1 == "//" && $2 == "+heroku" && $3 == "goVersion" ) { print $4; exit; } }' ${goMOD}
```

Reply:
Good catch.